pyalgotrading/algobulls/connection.py

```diff
@@ -79,7 +78,7 @@ def create_strategy(self, strategy, overwrite=False, abc_version=None):
 
         if abc_version is None:
             if isinstance(versions_supported, list):
-                _abc_version = strategy.versions_supported()[0].value       # Take the first version
+                _abc_version = strategy.versions_supported()[0].value  # Take the first version
```

Comment:
the TODO below this line, kindly check if it has been addressed and then remove it

Reply:
@pushpakdagade Please look into this